### ISSUE TITLE
YouTube fails to play songs • 1.20.1

### ISSUE THREAD

**Issue by ahykono** [HooferDevelops/url-music-discs#9]
https://www.youtube.com/watch?v=oGi8PDit6cQ should explain the clear issue

**Comment by HaipaDev** [comment]
Yeah I have also just tested the same scenario with cdn links working but youtube not, could be some incorrect yt-dlp settings or smth

**Comment by Plompi** [comment]
on which operating system did you try to use it? it should be fixed on linux with #21

**Comment by HaipaDev** [comment]
> on which operating system did you try to use it? it should be fixed on linux with #21 
For me Windows 11

**Comment by Plompi** [comment]
Thats weird, it should work perfectly fine on Windows, which version of the mod did you use? Did you use it on a singleplayerworld or on a Server (do you know the OS of the Server?)

**Comment by HaipaDev** [comment]
> Thats weird, it should work perfectly fine on Windows, which version of the mod did you use? Did you use it on a singleplayerworld or on a Server (do you know the OS of the Server?)

yeah, I was running it locally singleplayer, version 1.1.0

**Comment by Plompi** [comment]
soon there should be a major release 1.3.0 which fixes a lot of things, i hope it works at that point for you, if not another Issue with more details regarding the problem would be great 👍🏻

**Comment by Q7Sa** [comment]
Same Issue, I'm on Windows 11 and had a friend who has Windows 10 (NT) and had the same issue + same log output.

**Comment by xMdb** [comment]
> soon there should be a major release 1.3.0 which fixes a lot of things, i hope it works at that point for you, if not another Issue with more details regarding the problem would be great 👍🏻

any update on this release?

**Comment by HooferDevelops** [comment]
> > soon there should be a major release 1.3.0 which fixes a lot of things, i hope it works at that point for you, if not another Issue with more details regarding the problem would be great 👍🏻
> 
> any update on this release?

https://github.com/Plompi/VinURL
